website/search_migration/__init__.py

```diff
@@ -741,3 +695,34 @@
   AND id <= {page_end}
 LIMIT 1;
 """
+
+JSON_DELETE_PREPRINTS_SQL = """
```

Comment:
Would remove preprints from index that should no longer appear (for example, the preprint has been marked as private).